x-pack/legacy/plugins/apm/server/lib/errors/get_error_groups.ts

```diff
@@ -30,8 +30,8 @@ export async function getErrorGroups({
   setup
 }: {
   serviceName: string;
-  sortField: string;
-  sortDirection: string;
+  sortField?: string;
+  sortDirection?: string;
```

Comment:
Was this a mistake you caught? Or why the change?

Reply:
They are actually optional, if I look at both the client code and the joi schema, but the type casts converts them to required props.